src/wm_error.c

```diff
@@ -113,7 +113,7 @@ void _WM_ERROR(const char * func, unsigned int lne, int wmerno,
         "Unable to load",
         "Unable to open",
         "Unable to read",
-        "Invalid or Unsuported file format",
+        "Invalid or Unsupported file format",
```

Comment:
@sezero and @chrisisonwildcode Why is this in wildmidi twice?

I mean the following function:
static const char *errors[WM_ERR_MAX+1]